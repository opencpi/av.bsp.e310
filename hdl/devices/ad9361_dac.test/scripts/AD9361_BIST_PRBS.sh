../../ad9361_adc.test/scripts/AD9361_BIST_PRBS.sh