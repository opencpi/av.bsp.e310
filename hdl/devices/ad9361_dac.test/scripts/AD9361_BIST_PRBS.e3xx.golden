../../ad9361_adc.test/scripts/AD9361_BIST_PRBS.e3xx.golden